src/components/styles/createTheme.js

```diff
@@ -30,7 +30,21 @@ export default function createTheme(options = {}) {
   const theme = createMuiTheme(
     {
       breakpoints,
-      palette,
+      palette: {
+        primary: {
+          light: '#032C51',
+          main: '#26252D',
+          dark: '#000000',
+        },
+        secondary: {
+          light: '#F8F8F8',
+          main: '#E9E9E9',
+          dark: '#C90000',
+        },
+        text: {
+          secondary: '#C90000',
+        },
+      },
```

Comment:
Great! Maybe we need to adjust the pallette a little when we are building the FeaturedText-block. But it's all good for now!